subsys/debug/tracing/tracing_backend_posix.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019 Intel corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <kernel.h>
+#include <assert.h>
+#include <tracing_backend.h>
+
+static void *out_stream;
+
+static void tracing_backend_posix_init(void)
+{
+	out_stream =
+		(void *)fopen(CONFIG_TRACING_BACKEND_POSIX_FILE_NAME, "wb");
```

Comment:
Why are you using a Kconfig here and not a command line option to the exe like it was done in subsys/debug/tracing/ctf/bottoms/posix/ctf_bottom.c?

Reply:
@nashif I think it's more easy to understand and save code size.

Reply:
I do not think it is easy to understand, it makes thing a bit more complex and incompatible with how it is done now, also, code size is not an issue with native_posix.

Reply:
@nashif you are right, we don’t need care about code size for native posix. And this   PR get file name at compile name, while current existing one gets posix file at run time, that’s the difference, I prefer the widely used Kconfig in Zephyr project.

Reply:
@nashif updated